src/documentation/browserdialog.cpp

```diff
@@ -37,6 +37,23 @@ BrowserDialog::BrowserDialog(QWidget *parent) :
 BrowserDialog::~BrowserDialog()
 {
     delete ui;
+    delete mConnectionManager;
+
+    QDir dir = QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation);
+
+    bool result;
+    result = dir.cd("PairStorm");
+    result = dir.cd("temp");
+
```

Comment:
results of evaluation in line 45 are overwritten by evaluation in line 46
if result has to be true if both statements are true than use:
result &= dir.cd("...");

Reply:
![image](https://user-images.githubusercontent.com/18701555/63168703-8b0e0280-c03d-11e9-925c-f92e66d3728f.png)
I have already **changed & tested** this.
It works fine :)